react-frontend/src/components/Resources/guides.js

```diff
@@ -23,11 +23,11 @@ function Guides() {
             </Col>
             <Col sm={12} md={6}>
               <InfoCard
-                title={'Video Communication Platforms'}
-                description={
-                  'When you can’t meet with your coworkers or clients face-to-face, communicating with them by video can be the next best thing. There are many options available for video communication platforms or tools, many of which can also be used for instant messaging or chat, screen sharing and transferring files.'
-                }
-                // linkText={'More'}
+                isRouteCard
+                title="Video Communication Platforms"
+                description="When you can’t meet with your coworkers or clients face-to-face, communicating with them by video can be the next best thing. There are many options available for video communication platforms or tools, many of which can also be used for instant messaging or chat, screen sharing and transferring files."
```

Comment:
can the content be abstracted away into an object? This will make integrating with strapi even that more seemless

Reply:
Maybe?  I think the cards are going to need to be refactored anyway

Reply:
totally for make it work then make it right